cypress/integration/manualTests/basic-enduser.ts

```diff
@@ -100,14 +100,14 @@ describe('Test logged out home page', () => {
   it('home page search bar works', () => {
     cy.visit('/');
     cy.get('input[id=searchBar]').type('test{enter}');
-    cy.url().should('include', '/search?search=test');
+    cy.url().should('include', '/search?entryType=workflows&search=test');
```

Comment:
Thought, this is a test so not strictly responsible but got me thinking,
what happens with an old search that someone had bookmarked that doesn't specify an entry type?

Reply:
defaults to workflow with possibly weird facets selected

Reply:
I'm fine with defaults to workflows. Why weird facets selected though?

Reply:
if you previously selected tool-specific facets and have a link for that, it'll now default to looking for a workflow with that facet selected...which won't yield any results anymore

Reply:
Oh ok, I think I can live with that